sdk/storage/Azure.Storage.Blobs/src/LeaseClient.cs

```diff
@@ -274,7 +274,7 @@ private void EnsureClient()
                             ifMatch: httpAccessConditions?.IfMatch,
                             ifNoneMatch: httpAccessConditions?.IfNoneMatch,
                             async: async,
-                            operationName: "Azure.Storage.Blobs.Specialized.LeaseClient.Acquire",
+                            operationName: Constants.Blob.Lease.AcquireOperationName,
```

Comment:
I'm kind of on the fence about whether these should be pulled into `Constants` or left here using `nameof(...)` so they'll be automatically updated when refactoring.  Namespaces get kind of annoying with `nameof` (i.e., we'd have to do this particular string by `$"{nameof(Azure)}.{nameof(Azure.Storage)}.{nameof(Azure.Storage.Blobs)}.{nameof(Azure.Storage.Blobs.Specialized)}.{nameof(LeaseClient)}.{nameof(Acquire)}"`).

Anyone else have thoughts about this?  I could go either way -- and I'm also fine checking this in for now and doing that work separately if we want to since this PR is already a nice improvement.

Reply:
I kind of prefer the nameof(...) option better, but don't have strong feelings.

Reply:
Okay - let's keep things as they are in this PR for Preview 2 and I added an issue to switch to `nameof` for Preview 3 at #7105.